### PR TITLE
#166880069 Update travisCI to run tests.

### PR DESCRIPTION
**Pull request by frankopkusianwar**
#### What does this PR do?

Update travisCI to run tests

#### Description of Task to be completed?

- enable github repository to run tests on travisCI
- add .travis.yml file for configurations

#### How should this be manually tested?
Go to this branch on github
There should be a badge showing passing tests
click on the bagde to view the log on traviCI

### Screenshots.
trav

#### What are the relevant pivotal tracker stories?

[#166880069](https://www.pivotaltracker.com/story/show/166880069)

### CODE REVIEW

**Review by richien — APPROVED** [review]
Great work @frankopkusianwar

**Review by sebbss — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # ah-frontend-space
 [![Maintainability](https://api.codeclimate.com/v1/badges/9b2f555288730e8a26b4/maintainability)](https://codeclimate.com/github/andela/ah-frontend-space/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/9b2f555288730e8a26b4/test_coverage)](https://codeclimate.com/github/andela/ah-frontend-space/test_coverage)
\ No newline at end of file
+[![Test Coverage](https://api.codeclimate.com/v1/badges/9b2f555288730e8a26b4/test_coverage)](https://codeclimate.com/github/andela/ah-frontend-space/test_coverage)
+[![Build Status](https://travis-ci.org/andela/ah-frontend-space.svg?branch=ch-update-travisCI-166880069)](https://travis-ci.org/andela/ah-frontend-space)
```
Comment (MuhanguziDavid) [inline README.md]: Nice work @frankopkusianwar , please ensure that the `Travis CI` badge points to the `develop` branch, not the `ch-update-travisCI-166880069` branch.

Comment (frankopkusianwar) [reply README.md]: @davi this has been resolved

**Pull request merged**